.changeset/deep-mangos-stand.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/datastore": minor
+"@fluid-private/test-end-to-end-tests": minor
```

Comment:
This package should not be part of the changet because it's private.

Reply:
Fixed